standards/documentation.md

```diff
@@ -0,0 +1,54 @@
+# Documentation Standards
+
+All coding projects MUST include and maintain minimum documentation standards.
+
+## Minimum Standards
+
+Applications MUST include a README
```

Comment:
I feel that we (devs) should take ownership and set our standards high to document our Apps, Services, Lambdas as much as possible. Even if the codebase doesn't do much, it should still contain at bare minimum 1) Summary/Responsibilities 2) Ownership/Point of Contact

Some apps 'could' include much more. That depends on the complexity of the codebase, others might simply do one thing only but should specify 1) What the app is intended to do/solve and 2) Who to contact if an issue/question/enhancement is desired...

This will get us to shift from using other documentation platforms such as Confluence, Internal Sites, Google Docs, Google Presentations, Slack Documents, etc...